src/test/kotlin/no/nav/syfo/ikkeaktuell/api/IkkeAktuellApiSpek.kt

```diff
@@ -105,6 +105,7 @@ class IkkeAktuellApiSpek : Spek({
                         kafkaDialogmoteKandidatEndring.arsak shouldBeEqualTo DialogmotekandidatEndringArsak.IKKE_AKTUELL.name
                         kafkaDialogmoteKandidatEndring.kandidat shouldBeEqualTo false
                         kafkaDialogmoteKandidatEndring.unntakArsak shouldBe null
+                        kafkaDialogmoteKandidatEndring.veilederident shouldBe null
```

Comment:
Ikke en del av denne oppgaven, men vi burde kanskje hatt `veilederIdent` ved ikke aktuell også? Hvis vi ikke så burde man kanskje kalt denne unntakVeilederIdent i stedet :thinking:

Reply:
Så vidt jeg klarte å se er det bare for unntak vi persisterer dette (men det kan være vi kunne lagt den med likevel). 

Kan høre med Lars, men oppfattet det slik at det var for unntak dette var mest interessant, og hvis det stemmer kan vi jo renavne slik du foreslår.

Reply:
Tror vi persisterer det for `IkkeAktuell` også, men ser ut som det må litt mer endringer til for å hente det ut der hvor vi sender på kafka. Men ja, hør med Lars, og hvis det ikke er et behov så er det best å rename :+1:

Reply:
Lars mener vi går for veilederident bare for unntak (i første omgang i hvert fall). Renavnet til `unntakVeilederident`.